install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -144,6 +144,8 @@ data:
               fieldPath: metadata.name
         - name: ISTIO_META_INTERCEPTION_MODE
           value: {{ "[[ or (index .ObjectMeta.Annotations \"sidecar.istio.io/interceptionMode\") .ProxyConfig.InterceptionMode.String ]]" }}
+        - name: ISTIO_META_NETWORK
```

Comment:
The update is good to me, but we intend to keep this file consistent with the `sidecar-injector-configmap.yaml` in `istio` chart, so here the question is why not update `istio` chart as well?

Reply:
Make sense. Will update istio too.